tests/features/demo/demo_june.feature

```diff
@@ -4,18 +4,19 @@ Feature: June 2016 demo
   I should be able to interact with the website and manage content.
 
   Scenario: Manage collection and view collection scenarios.
-    Given collections:
-      | title          | description                           | logo     | moderation |
-      | S.H.I.E.L.D.   | Well, they are mostly flying around.  | logo.png | yes        |
-      | x-Men          | Based on Proffessor Xavier's mantion. | logo.png | no         |
-      | Avengers       | Based on Tony stark's tower.          | logo.png | yes        |
-      | Fantastic four | Based on Reed Richard's tower.        | logo.png | yes        |
     And solutions:
```

Comment:
Yeah sorry... :/ We have to have the solutions created in order to affiliate them to collections and I changed this to a lot of tests! swapped them...